client/src/modules/stock/requisition/modals/search.modal.html

```diff
@@ -38,6 +38,34 @@
             </div>
           </div>
 
+          <div>
+            <label class="control-label" translate>FORM.LABELS.STATUS</label>
+            <div class="row">
+              <div ng-repeat ="r in $ctrl.requisitionStatus | orderBy:'plainText'">
+                <span class="col-md-6">
+                  <span ng-if="$index %2 == 0">
+                    <div class="checkbox">
+                      <label>
+                        <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
+                        <span ng-class="r.class_style" translate> {{ r.plainText }} </span>
+                      </label>
+                    </div>
+                  </span>
+                </span>
+                <span class="col-md-6">
+                  <span ng-if="$index %2 != 0">
+                    <div class="checkbox">
+                      <label>
+                        <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
+                        <span ng-class="r.class_style" translate> {{ r.plainText }} </span>
+                      </label>
+                    </div>
+                  </span>
+                </span>
+              </div>
+            </div>
+          </div>   
```

Comment:
You can also write (if the even or odd doesn't matter) : 
```html
<div class="row">
  <div ng-repeat ="r in $ctrl.requisitionStatus | orderBy:'plainText'" class="col-md-6">
    <div class="checkbox">
      <label>
        <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
        <span ng-class="r.class_style" translate> {{ r.plainText }} </span>
      </label>
    </div>
  </div>
</div>
```

Reply:
I will do